spring-boot-admin-server-ui/modules/applications-details/css/module.css

```diff
@@ -0,0 +1,21 @@
+dl.health-status {
+    margin-top: 0;
+    line-height: 20px;
+}
+
+dl.health-status dt {
+    border-bottom: 1px solid #ddd;
+    background-color: #f9f9f9;
+    padding: 8px;
+    text-transform: capitalize;
+}
+
+dl.health-status dl.health-status {
+    margin-top: 20px !important;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://www.codacy.com/app/joshiste/spring-boot-admin/file/2593395628/issues/source?bid=3175464&fileBranchId=3306528#l14)